mkdocs.yml

```diff
@@ -0,0 +1,38 @@
+site_name: Cosmotech Acceleration Librairy Docs
+theme:
+  name: material
+  font:
+    code: Fira Mono
+  palette:
+    scheme: slate
+    primary: orange
+  features:
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - navigation.expand
```

Comment:
With the large quantity of automatically created files we could remove the `navigation.expand` to avoid having the full nav opened by default. (examples of effect [here](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion) )